atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -309,14 +309,8 @@ private void registerMetrics(GenericObjectPool<CassandraClient> pool) {
         registerPoolMetric("meanBorrowWaitTimeMillis", pool::getMeanBorrowWaitTimeMillis);
         registerPoolMetric("numIdle", pool::getNumIdle);
         registerPoolMetric("numActive", pool::getNumActive);
```

Comment:
I think this is a good change. Just wondering if we could basically keep all metrics in memory and then use your filtering infra to report outliers?

Reply:
Yep, we could. This was intended to be the 'pain free' or 'risk free' version of this PR, that's definitely something that we could do (just more complex)